lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -34,6 +36,8 @@ def commands
       "db_connect"    => "Connect to an existing database",
       "db_disconnect" => "Disconnect from the current database instance",
       "db_status"     => "Show the current database status",
+      "db_save"       => "Save the current database connection so it is re-used on startup",
+      "db_remove"     => "Remove the saved entry for the specified data service."
```

Comment:
Minor: other help entries don't end with a period. Consider rewording to something along the lines of "Remove the saved data service entry for the specified data service". See also: the `cmd_db_remove_help` description.